peerberrypy/api.py

```diff
@@ -422,7 +422,7 @@ def get_mass_investments(
             self,
             quantity: int = 100000000000,
             sort: str = 'invested_amount',
-            countries: list = None,
+            countries: Optional[list[str]] = None,
```

Comment:
Instead of having `list[str]`, it's better to have `List[str]` to ensure backward compatibility with older Python versions. You can use the List from the typing module.